### DISCUSSION TITLE
How to handle 500 errors from /server/api

### DISCUSSION THREAD

**Discussion by turhobr** [nuxt/nuxt#22921] [Questions]
Hello,
When something goes wrong in /server/api and it's not in try-catch block 500 error is returned (https://nuxt.com/docs/guide/directory-structure/server#error-handling) but in response body it returns all information (like message and stacktrace) which can be potentially vulnerable. I wasn't able to catch these errors in any way. Either any of hooks or error.vue in root. Is there any chance to catch these errors?

Thanks

**Comment by xjccc** [comment]
https://nuxt.com/docs/api/composables/use-error

useError in error.vue?
